manifests/y/Yandex/Browser/23.1.3.952/Yandex.Browser.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Yandex.Browser

```

```suggestion
PackageIdentifier: Yandex.Yandex
```
WinGet generally follows the naming scheme `Publisher.PackageName` for the PackageID.